### PR TITLE
Manage record workflow

### PR DESCRIPTION
**Pull request by christophermmeyer**
Adding in the manage_records workflow to the Community folder. This contains a set of endpoint to handle managing various record types

### CODE REVIEW

**Comment by nikhiljangi** [comment]
DS store file (Community/Service Requests/.DS_Store)  appears to be accidentally added to the pull request. Please remove the same.

**Comment by sharma-anshul** [comment]
Hey @christophermmeyer this pull request should only contain files related to the Manage Records workflow, but it seems to contain commits related to Service Requests as well. Those files would have to be removed before we can merge this in.

**Comment by christophermmeyer** [comment]
@sharma-anshul and @nikhiljangi All the changes mentioned should be taken care of now!

File: Community/manage_records/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2017 BlueCat Networks. All rights reserved.
```
Comment (delmeherbert) [inline Community/manage_records/__init__.py]: Please change date to 2019. There is another instance of 2017 in manage_records.py

Comment (christophermmeyer) [reply Community/manage_records/__init__.py]: @delmeherbert Updated to change to 2019

**Comment by christophermmeyer** [comment]
Closing this out and redoing another pull request to fix this weirdness.

**Pull request closed without merging**